src/components/views/settings/DevicesPanel.js

```diff
@@ -163,7 +174,7 @@ export default class DevicesPanel extends React.Component {
         });
     }
 
-    _makeDeleteRequest(auth) {
+    _makeSignOutRequest(auth) {
```

Comment:
This should probably stay named as delete to form the association to the endpoint rather than the button